core/scripts/keystone/src/05_deploy_initialize_capabilities_registry.go

```diff
@@ -26,110 +26,134 @@ import (
 )
 
 type peer struct {
-	PeerID string
-	Signer string
+	PeerID              string
+	Signer              string
+	EncryptionPublicKey string
 }
 
 var (
 	workflowDonPeers = []peer{
 		{
-			PeerID: "12D3KooWBCF1XT5Wi8FzfgNCqRL76Swv8TRU3TiD4QiJm8NMNX7N",
-			Signer: "0x9639dCc7D0ca4468B5f684ef89F12F0B365c9F6d",
+			PeerID:              "12D3KooWBCF1XT5Wi8FzfgNCqRL76Swv8TRU3TiD4QiJm8NMNX7N",
+			Signer:              "0x9639dCc7D0ca4468B5f684ef89F12F0B365c9F6d",
+			EncryptionPublicKey: "0xe7f44e3eedf3527199efec7334183b5384ba0e7c7c57b390b63a3de5a10cd53c",
```

Comment:
Thanks! Do we have the private keys stored somewhere?

Reply:
These are random values.